NICE.Identity.Management/ClientApp/src/views/Organisation/Organisation.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from "react";
+import { RouteComponentProps, Link } from "react-router-dom";
+import { StaticContext } from "react-router";
+
+import { OrganisationType, UserType } from "../../models/types";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { PageHeader } from "@nice-digital/nds-page-header";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+import { Alert } from "@nice-digital/nds-alert";
+
+import { fetchData } from "../../helpers/fetchData";
+import { isDataError } from "../../helpers/isDataError";
+import { Endpoints } from "../../data/endpoints";
+
+import { Button } from "@nice-digital/nds-button";
+
+import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
+import styles from "./Organisation.module.scss";
+
+import { UserStatus } from "../../components/UserStatus/UserStatus";
+import { ToFormattedDateString } from "../../helpers/dateHelpers";
+
+type TParams = { id: string };
+
+type LocationState = {
+	hasBeenEdited: boolean;
+};
+
+type OrganisationProps = Record<string, unknown> &
+	RouteComponentProps<TParams, StaticContext, LocationState>;
+
+type OrganisationState = {
+	organisation: OrganisationType;
+	users: Array<UserType>;
+	error?: Error;
+	redirect: boolean;
+	isLoading: boolean;
+};
+
+export class Organisation extends Component<OrganisationProps, OrganisationState> {
+		constructor(props: OrganisationProps) {
+			super(props);
+	
+			this.state = {
+				organisation: {} as OrganisationType,
+				users: [],
+				redirect: false,
+				isLoading: true,
+			};
+		}
+
+		async componentDidMount(): Promise<void> {
+			this.setState({ isLoading: true });
+	
+			const organisation = await fetchData(Endpoints.organisation(this.props.match.params.id));
+			const users = await fetchData(Endpoints.usersByOrganisation(this.props.match.params.id));
+
+			if (isDataError(users)) {
+				this.setState({ error: users });
+			}
```

Comment:
I think we need to check organisation against isDataError too here like we do for users.